archive/p/perl/README.md

```diff
@@ -6,6 +6,7 @@ Welcome to Sample Programs in Perl!
 
 - [Hello World in Perl](https://therenegadecoder.com/code/hello-world-in-perl/)
```

Comment:
Looks like this pulled in your changes from Perl as well which is fine. Can you make the same README changes here? 👍

Reply:
Shoot! I didn't mean to do that, sorry. Updated the Perl links just now though.